.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+// Config documentation:
+// https://containers.dev/implementors/json_reference/
+{
```

Comment:
🐼 

Maybe worth adding the VS code settings if adding a Dev container configuration?

Reply:
Done 👍